code/presentation/mb-client/src/navigation/home-navigator.tsx

```diff
@@ -0,0 +1,68 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import * as React from 'react';
+
+import { supabase } from '@/core/supabase'; // Import supabase instance
+import { HomeScreen } from '@/screens/home';
+import AudioScreen from '@/screens/home/audio-screen';
+import FindMasjid from '@/screens/home/find-masjid';
+import MasjidInfo from '@/screens/home/masjid-info';
+import MasjidScreen from '@/screens/home/masjid-screen';
+import NamazTimingsScreen from '@/screens/home/namaz-timings';
+import VideoScreen from '@/screens/home/video-screen';
+
+export type HomeStackParamList = {
+  HomeScreen: undefined;
+  FindMasjid: undefined;
+  MasjidScreen: { selectedMasjidId: number };
+  TestScreen: undefined;
+  NamazTimingsScreen: { selectedMasjidId: number };
+  AudioScreen: undefined;
+  VideoScreen: undefined;
+  MasjidInfo: { selectedMasjidId: number };
+  FindMasjid1: undefined;
+};
+
+const Stack = createNativeStackNavigator<HomeStackParamList>();
+
+export const HomeNavigator = () => {
+  const [subscribedMasjids, setSubscribedMasjids] = React.useState<any[]>([]);
+
+  const fetchSubscribedMasjids = async () => {
+    try {
+      const { data, error } = await supabase.rpc('get_subscribed_masjids');
```

Comment:
can use already defined method in supabase.ts

Reply:
Used the getsubscribed function method from supabase